quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testExportHelpAll.unix.approved.txt

```diff
@@ -115,6 +200,12 @@ Logging:
                        categories and their levels. For the root category, you don't need to
                        specify a category. Default: info.
 
+Security (Preview):
```

Comment:
Do make sense security options on import/export?

Reply:
To my understanding FIPS is at the moment mostly (only?) about a crypto provider. I wouldn't be sure if this might affect any provider we use during import/export, therefore would keep it.

Reply:
I don't think it does impact. But @mposolda may confirm.